examples/feature_demo/gltf_animations.py

```diff
@@ -37,7 +37,7 @@
 from rendercanvas.auto import RenderCanvas, loop
 
 from wgpu.utils.imgui import ImguiRenderer
-from imgui_bundle import imgui  # type: ignore
+from imgui_bundle import imgui, hello_imgui, icons_fontawesome_6  # type: ignore
```

Comment:
You should use `icons_fontawesome_4` here, since hello imgui will load this version by default